demos/instance_segmentation_demo/python/instance_segmentation_demo.py

```diff
@@ -18,205 +18,229 @@
+                              'If 0 is set, all frames are stored.')
+    io_args.add_argument('--no_show', action='store_true',
+                         help="Optional. Don't show output.")
+    io_args.add_argument('--output_resolution', default=None, type=resolution,
+                         help='Optional. Specify the maximum output window resolution '
+                              'in (width x height) format. Example: 1280x720. '
+                              'Input frame size used by default.')
+    io_args.add_argument('-u', '--utilization_monitors',
+                         help='Optional. List of monitors to show initially.')
+
+    debug_args = parser.add_argument_group('Debug options')
+    debug_args.add_argument('-r', '--raw_output_message', action='store_true',
+                             help='Optional. Output inference results raw values showing.')
     return parser
 
 
+def get_model(model_adapter, configuration):
+    inputs = model_adapter.get_input_layers()
+    outputs = model_adapter.get_output_layers()
+    if len(inputs) == 1 and len(outputs) == 4 and 'proto' in outputs.keys():
+        return YolactModel(model_adapter, configuration)
+    return MaskRCNNModel(model_adapter, configuration)
+
+
+def print_raw_results(boxes, classes, scores, frame_id):
+    log.debug('  -------------------------- Frame # {} --------------------------  '.format(frame_id))
+    log.debug('  Class ID | Confidence |     XMIN |     YMIN |     XMAX |     YMAX ')
+    for box, cls, score in zip(boxes, classes, scores):
+        log.debug('{:>10} | {:>10f} | {:>8.2f} | {:>8.2f} | {:>8.2f} | {:>8.2f} '.format(cls, score, *box))
+
+
 def main():
     args = build_argparser().parse_args()
 
     cap = open_images_capture(args.input, args.loop)
 
-    with open(args.labels, 'rt') as labels_file:
-        class_labels = labels_file.read().splitlines()
-        assert len(class_labels), 'The file with class labels is empty'
-
-    # Plugin initialization for specified device and load extensions library if specified.
-    log.info('OpenVINO Inference Engine')
-    log.info('\tbuild: {}'.format(get_version()))
-    core = Core()
-    if args.cpu_extension and 'CPU' in args.device:
-        core.add_extension(args.cpu_extension, 'CPU')
-
-    # Read IR
-    log.info('Reading model {}'.format(args.model))
-    model = core.read_model(args.model)
-    image_input, image_info_input, (n, c, h, w), model_type, output_names, postprocessor = check_model(model)
-    args.no_keep_aspect_ratio = model_type == 'yolact' or args.no_keep_aspect_ratio
-
-    compiled_model = core.compile_model(model, args.device)
-    infer_request = compiled_model.create_infer_request()
-    log.info('The model {} is loaded to {}'.format(args.model, args.device))
-
-    if args.no_track:
-        tracker = None
-    else:
-        tracker = StaticIOUTracker()
-
-    if args.delay:
-        delay = args.delay
-    else:
-        delay = int(cap.get_type() in ('VIDEO', 'CAMERA'))
```

Comment:
Without` delay` it's hard to see the result on images, so I suggest leave it in code, without allowing modifying it from cmd:
https://github.com/openvinotoolkit/open_model_zoo/blob/0d23ffa744c6c6dabec6a3afeac3a023d11b946c/demos/classification_demo/python/classification_demo.py#L158

Reply:
Most of Python demos support folder with images as input, so the delay handling should be introduced in all demos, theoretically, since the images are changed quickly in this case. 
I would prefer to unify Python model API demos at the same time in order to align with classification_demo.py, and investigate this option in more details, if you agree with it

Reply:
ok